avocado/utils/cpu.py

```diff
@@ -188,3 +189,56 @@ def offline(cpu):
     if _get_cpu_status(cpu):
         return 1
     return 0
+
+
+def get_cpu_idle():
+    """
+    Get current cpu idle values
+    """
+    cpus_list = cpu_online_list()
```

Comment:
Actually I only noticed this now. This can break whole lot of assumptions when disabling/enabling processors between setting the states. Do you have some very significant reason not to include offline cpus? Imagine a situation where you disable 3rd CPU, get state, change it a bit, enable 3rd cpu and reset the state? Instead of cpu 4 it'd revert the statuses of the 3rd CPU and all following CPUs will suffer the same issue.

Reply:
Sure, got you, I will rework on it